apps/aecore/src/aec_conductor.erl

```diff
@@ -228,6 +228,7 @@ handle_call({post_block, Block},_From, State) ->
 handle_call(stop_mining,_From, State = #state{ consensus = Cons }) ->
     epoch_mining:info("Mining stopped"),
     [ aec_tx_pool:garbage_collect() || is_record(Cons, consensus) andalso Cons#consensus.leader ],
+    aec_block_generator:stop_generation(),
```

Comment:
Yay, I noticed while analyzing some test logs the other day that we kept generating blocks after shutting down mining - good fix!

Reply:
👌